lib/fizzy/execute.cpp

```diff
@@ -489,10 +489,13 @@ std::optional<uint32_t> find_export(const Module& module, ExternalKind kind, std
     return (it != module.exportsec.end() ? std::make_optional(it->index) : std::nullopt);
 }
 
-inline uint64_t& get_local(std::vector<uint64_t>& locals, uint32_t idx) noexcept
+inline uint64_t& get_local(
+    std::vector<uint64_t>& args, std::vector<uint64_t>& locals, uint32_t idx) noexcept
 {
-    assert(idx <= locals.size());
-    return locals[idx];
+    if (idx < args.size())
```

Comment:
Without making any further optimisations, isn't this making it slower? Previously we copied arguments once.

Reply:
Much slower. See description.